3754/1/libs/take_data.rb

```diff
@@ -0,0 +1,38 @@
+require 'open-uri'
+# class for taking data about gem
+class TakeData
```

Comment:
so you can have DataFetcher class which parse data and initialize an object like GemStat which then collected to GemStatCollection object which is like storage